docs/configuration/miscellaneous.md

```diff
@@ -106,6 +106,16 @@ By default, NetBox will prevent the creation of duplicate prefixes and IP addres
 
 ---
 
+## EVENTS_PIPELINE
+
+!!! tip "Dynamic Configuration Parameter"
+
+Default: ['extras.events.process_event_queue',]
+
+NetBox will call functions listed here for events (create, update, delete) on models as well as when custom EventRules are fired.
```

Comment:
The description should specify passing a list of dotted paths (rather than the functions themselves).